pyproject.toml

```
@@ -6,16 +6,16 @@ authors = ["CTDS UChicago <cdis@uchicago.edu>"]
 license = "Apache-2.0"
 
 [tool.poetry.dependencies]
-python = "^3.7"
+python = "^3.9"
 fastapi = "^0.65.2"
-uvicorn = {extras = ["standard"], version="^0.11.2"}
+uvicorn = {extras = ["standard"], version="^0.18.2,<1.0.0"}
```

```suggestion
uvicorn = {extras = ["standard"], version="^0.18.2"}
```
The [caret by itself](https://python-poetry.org/docs/dependency-specification/#caret-requirements) should be enough to indicate`<0.19.0`.